specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/examples/Job_CreateorUpdate.json

```diff
@@ -2,7 +2,7 @@
   "parameters": {
     "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
     "resourceGroupName": "rg",
-    "jobName": "testcontainerAppsJob0",
+    "jobName": "testcontainerappsjob0",
```

Comment:
Lets fix the same for container apps

Reply:
Updated alll the `ContainerApps_&' test cases.